versioned_docs/version-5/url.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Share machines using their share URL
+---
+
+You can share machines with your team or publicly with the machine URL.
+
+Your machine will only be viewable if:
+
+- the project visibility is **public**
+- the project visibility is **unlisted**
+- the project visibility is **private**, and the viewer is also a member of your team
+
+Private machines stored in My Projects will only be viewable by you.
+
+Read [how to change a project’s visibility settings](projects.mdx#change-a-projects-visibility).
+
+## Copy the URL from the browser address bar
+
+You can copy the machine URL from the browser address bar, which will share the machine in your current view and version.
+
+## Copy the URL from the Share modal
+
+1. Use the **Share** button in Stately Studio’s top bar to open the Share dialog.
```

Comment:
Should we add an image to help visualize this?

Reply:
> Should we add an image to help visualize this?

I’m holding off right now because of the colour changes in blocks. But I’ll add a placeholder todo!